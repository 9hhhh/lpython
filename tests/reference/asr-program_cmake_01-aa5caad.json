{
    "basename": "asr-program_cmake_01-aa5caad",
    "cmd": "lfortran --show-asr {infile} -o {outfile}",
    "infile": "tests/../integration_tests/program_cmake_01.f90",
    "infile_hash": "0e854d9b2ed613d6193d78290dc20a45ec66f60ab17e1c0229c3febe",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-program_cmake_01-aa5caad.stdout",
    "stdout_hash": "2332d4760516d93cb4953455742b7b4e3481932fdf2da2f7ad4499ee",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}