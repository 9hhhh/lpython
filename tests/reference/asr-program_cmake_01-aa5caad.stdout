(TranslationUnit (SymbolTable c56cda33 {TESTFortran: (Program (SymbolTable 00000003 {}) TESTFortran [(Print () [(Str "'Hello'" (Character 8 []))])])}) [])
